assets/dev/js/editor/commands/internal/attach-preview.js

```diff
@@ -68,7 +68,7 @@ export class AttachPreview extends CommandInternalBaseBase {
 
 			document.$element
 				.addClass( 'elementor-edit-area-active' )
-				.removeClass( 'elementor-edit-area-preview elementor-editor-preview' );
+				.removeClass( 'elementor-editor-preview' );
```

Comment:
We need a method for CSS deprecation.

Reply:
It's impossible

Reply:
A post on dev blog + wait 3 versions is the minimum.